src/main/java/com/binance/dex/api/client/domain/broadcast/Issue.java

```diff
@@ -2,11 +2,11 @@
 
 public class Issue {
 
-    private String from;
-    private String name;
-    private String symbol;
-    private Long totalSupply;
-    private Boolean mintable;
+    protected String from;
+    protected String name;
+    protected String symbol;
+    protected Long totalSupply;
+    protected Boolean mintable;
```

Comment:
better remain those as 'private'. if you want to access from sub-class, then can use get** method

Reply:
sure. got it.